src/ttt/player.clj

```diff
@@ -0,0 +1,17 @@
+(ns ttt.player)
+
+(defn- player 
+  [marker]
+  {:marker marker})
+
+(def players [(player "X") (player "O")])
+
+(defn get-move
+  []
+  (read-string (read-line)))
+
+(defn switch-players
+  [[player-1 player-2]]
+  (if (= (:marker player-1) "X")
```

Comment:
this seems too specific, is there a way to do this without relying on one of the markers being an "X"?